src/main/scala/uk/gov/hmrc/play/bootstrap/http/HttpClient.scala

```diff
@@ -43,5 +43,5 @@ class DefaultHttpClient @Inject()(
     override def configuration: Configuration = config
   }.appName
 
-  override val hooks = Seq(AuditingHook)
+  override val hooks: Seq[HttpHook] = Seq(AuditingHook)
```

Comment:
declaring type here helps in overriding this field in the subclasses , for more clarification, please talk to me
 @sureshhmrc